stock_picking_batch_extended/data/batch_picking_actions_server.xml

```diff
@@ -26,10 +26,9 @@ action = env.ref('stock.action_report_picking').report_action(pickings)
             ref="stock_picking_batch.model_stock_picking_batch"
         />
         <field name="state">code</field>
-        <field
-            name="code"
-        >pickings = records.mapped('picking_ids').filtered(lambda p: p.state != 'cancel')
-action = env.ref('stock.action_report_delivery').report_action(pickings)
+        <field name="code">
+pickings = records.mapped('picking_ids').filtered(lambda p: p.state != 'cancel')
+action = env['ir.actions.report'].sudo()._get_report_from_name('stock.report_delivery').report_action(pickings)
```

Comment:
@BorisFL Are you sure about the sudo ? As it is done in _get_report_from_name()

Reply:
fixed